.github/workflows/server-docker-build-arm.yml

```
@@ -14,16 +14,17 @@ env:
 jobs:
   build_and_push:
     runs-on: [self-hosted, Linux, ARM64]
+    timeout-minutes: 45
     steps:
       - name: Checkout Code
         uses: actions/checkout@v2
```

actionlint (1.7.7)

the runner of "actions/checkout@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)